File: features/recharge/digitalproduct/src/main/java/com/tokopedia/digital/newcart/data/entity/response/voucher/Category.java
package com.tokopedia.digital.newcart.data.entity.response.voucher;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * @author anggaprasetiyo on 2/27/17.
 */

public class Category {

    @SerializedName("data")
    @Expose
    private RelationData data;

    public RelationData getData() {
        return data;
    }
}

File: features/merchant/official_store/src/main/java/com/tokopedia/officialstore/category/data/model/Category.kt
package com.tokopedia.officialstore.category.data.model

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
data class Category(
        @SerializedName("id")
        val categoryId: String = "",
        @SerializedName("name")
        val title: String = "",
        @SerializedName("imageUrl")
        val icon: String = "",
        @SerializedName("prefixUrl")
        val prefixUrl: String = "",
        @SerializedName("url")
        val slug: String = "",
        @SerializedName("fullUrl")
        val fullUrl: String = "",
        @SerializedName("categories")
        val categories: ArrayList<Int> = ArrayList(),
        @SerializedName("imageInactiveURL")
        val imageInactiveURL: String = ""
) : Parcelable